core/dbt/version.py

```diff
@@ -27,81 +51,175 @@ def get_latest_version(version_url: str = PYPI_VERSION_URL):
     return dbt.semver.VersionSpecifier.from_version_string(version_string)
 
 
-def get_installed_version():
-    return dbt.semver.VersionSpecifier.from_version_string(__version__)
+def _get_core_msg(installed, latest) -> Tuple[str, str]:
+    core_msg_lines, update_info = _get_core_msg_lines(installed, latest)
+    return (
+        _format_core_msg(core_msg_lines),
```

Comment:
Looks like this would just call the other two functions, should we consider consolidating three functions into one and just use comments to separate it into pieces?

Reply:
Totally! I think this was stripped down during refactoring, and now serves no real purpose.